### DISCUSSION TITLE
Supabase Auth links are broken

### DISCUSSION THREAD

**Discussion by arshamg** [supabase/supabase#15244] [General]
According to the status page a fix was rolled out, but we are still seeing broken links. Here's an example for our staging environment:

https://kkbwmckdgyhidisstuoq.supabase.co/auth/v1/verify?redirect_to=https://staging.ribbon.cool/&token=6af7524f1281b9bfd97c32798d3a830f818cabcd047b4c38477b5045&type=signup/email-verified

the redirect_to is encoded incorrectly.

**Comment by Cayllen** [comment]
Hey, is it still broken for you & how did you fix it? I have the same issue

**Reply by arshamg** [reply]
This turned out to be because we we're appending something to the links in the Supabase email template (templates are somewhere within the auth part of the dashboard). It worked previously because the links were encoded differently.
